__tests__/config/formats/typescript-export-definition.test.ts

```diff
@@ -0,0 +1,124 @@
+import {typescriptExportDefinition} from '../../../config/formats/typescript-export-defition'
+
+describe('Format: Tyepscript definitions', () => {
+  const dictionary = {
+    tokens: {
+      colors: {
+        red: {
+          $type: 'color',
+          value: '#FF0000'
+        },
+        yellow: {
+          value: '#FF0000'
+        }
+      },
+      size: {
+        default: {
+          value: 16,
+          $type: 'dimension'
+        },
+        // px: {
```

Comment:
Are you planning to remove these?

Reply:
No I am planning on re-enabling them once we move the size stuff over to this build process.